ISISComputingGroup/ConfigChecker
# Ticket 2959: Add check for simulation mode

See: https://github.com/ISISComputingGroup/IBEX/issues/2959

Added configuration and global checking for simulation flags.

To test:
* Ensure unit tests pass
* Run config-checker pointing at DEMO
* Confirm fails on simulation mode
* Run config-checker pointing at other instrument
* Confirm fails on simulation mode
